core/client/pom.xml

```diff
@@ -79,6 +79,11 @@
       <artifactId>alluxio-underfs-s3</artifactId>
       <version>${project.version}</version>
     </dependency>
+    <dependency>
+      <groupId>org.alluxio</groupId>
+      <artifactId>alluxio-underfs-swift</artifactId>
+      <version>${project.version}</version>
+    </dependency>
```

Comment:
@calvinjia yes, if i understand you correctly - then Swift module doesn't need any of Hadoop dependencies anymore. Without swift lines in the client - Spark can't work with Swift.